src/main/java/org/openmrs/maven/plugins/AbstractTask.java

```diff
@@ -62,6 +64,20 @@
     Wizard wizard;
 
     /**
+     * Interactive mode param
+     *
+     * @parameter expression="${interactiveMode}" default-value=true
+     */
+    String interactiveMode;
```

Comment:
Yeah, that's for testing, when enabled sdk doesn't stop on `yes/no` prompts, I will comment it better